release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -235,6 +235,30 @@ Limit Calculator] to estimate cluster limits for your environment.
 [id="ocp-4-4-developer-experience"]
 === Developer experience
 
+[id="ocp-4-4-automatic-image-pruning"]
+==== Automatic image pruning
+
+You can now enable automatic image pruning. This is not enabled by default;
+you will be notified of this option after installing or upgrading to
+{product-title} 4.4. This automation is managed by the Image Registry Operator, which
+creates a CronJob to run periodic image pruning.
+
+[id="ocp-4-4-build-objects-report-conditions-in-status"]
+==== Build objects report conditions in status
+
+Build conditions have been added for each existing {product-title} build phase.
+These conditions contain information about the build during its build lifecycle.
+You can also use commands like `oc wait` to wait for a specific build phase to
+be reached.
+
+[id="ocp-4-4-recreate-rollouts-for-image-registry"]
+==== Recreate rollouts for image registry
+
+You can now use the `Recreate` rollout strategy when deploying the image
+registry. This lets you use `ReadWriteOnce` persistent volumes, such as Elastic
```

Comment:
s/such as Elastic Block Storage on AWS/such as AWS Elastic Block Store/
(This is how we reference it [elsewhere](https://docs.openshift.com/container-platform/4.3/storage/persistent_storage/persistent-storage-aws.html) in the docs.)